gvr-jassimp/app/src/main/java/org/gearvrf/gvrjassimp/JassimpModelLoaderMain.java

```diff
@@ -32,6 +34,31 @@
 import org.gearvrf.utility.Log;
 
 import android.graphics.Color;
+class AssetListener implements IAssetEvents
+{
+    private GVRScene mScene;
+    
+    public AssetListener(GVRScene scene) {
+        mScene = scene;
+    }
+    
+    public void onModelLoaded(GVRContext context, GVRSceneObject model, String modelFile) { }
+    public void onTextureLoaded(GVRContext context, GVRTexture texture, String texFile) {}
+    public void onModelError(GVRContext context, String error, String modelFile) { }
+    public void onTextureError(GVRContext context, String error, String texFile) { }
+    public void onAssetLoaded(GVRContext context, GVRSceneObject model, String filePath, String errors)
+    {
+        if ((model != null) && (errors == null))
```

Comment:
Isn't "errors" supposed to be a "" in case of no error (see AssetEventListener.beginLoadAsset)? I'd say change the assetLoader to pass null per javadoc.